src/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>To Do List</title>
+</head>
+<body>
+  <main>
+    <div class="title">
+      <h3>Today's To Do</h3>
+      <img src="#" alt="Refresh Icon" class="refresh-icon">
```

Comment:
Image source not found

Reply:
The `src` attribute name `#` is replaced dynamically with javascript.